client/data/authorizations/resolvers.ts

```diff
@@ -64,6 +72,49 @@ export function* getAuthorizations( query: Query ): any {
 	yield updateAuthorizations( query, getMockedRows() );
 }
 
+export function* getAuthorization( id: string ): Generator< unknown > {
+	// TODO replace mocked implementation when server is ready
+	const randomAmount = () => {
+		return 1400 + Math.floor( Math.random() * 5000 );
+	};
+
+	const randomDate = dateI18n(
+		'M j, Y / g:iA',
+		moment.utc( new Date() ).local().toISOString()
+	);
+
+	const randomCaptureDate = dateI18n(
```

Comment:
Although mocked data here will eventually be removed, using static (not random) dates would help in writing predictable tests.

Reply:
I removed the mocked data in favor of using fixtures in [eaa5fba](https://github.com/Automattic/woocommerce-payments/pull/4676/commits/eaa5fbab64503b208bfa08223c547e90091d6d25) so I guess this comment no longer applies, please let me know otherwise @anu-rock

Reply:
It looks quite better now 👍

Now that the resolver returns empty array/object, what would be the best way to capture button functionality with some data?

Reply:
I'd suggest to import the mocked data from the authorizations fixture. In `client/data/authorizations/resolvers.ts`:

```
+import authorizationsFixture from '../../data/authorizations/test/authorizations.fixture.json';
 
export function* getAuthorizations( query: Query ): Generator< unknown > {
-       yield updateAuthorizations( query, [] );
+       yield updateAuthorizations( query, authorizationsFixture );
}
```